sentry-sidekiq/spec/sentry/sidekiq_spec.rb

```diff
@@ -223,7 +223,7 @@ def retry_last_failed_job
       expect(transaction.contexts.dig(:trace, :trace_id)).to be_a(String)
       expect(transaction.contexts.dig(:trace, :span_id)).to be_a(String)
       expect(transaction.contexts.dig(:trace, :status)).to eq("ok")
-      expect(transaction.contexts.dig(:trace, :op)).to eq("queue.sidekiq")
```

Comment:
@frederikspang Did this break sampling? In the [docs](https://docs.sentry.io/platforms/ruby/guides/sidekiq/configuration/sampling/#setting-a-sampling-function) you show that you can do 

```rb
case op
when /sidekiq/
  0.01
[...]
```

But this is always set to `queue.process` now so we are sampling a lot more than we should which is costing us money...

Was this intentional or am I misunderstanding something?

Reply:
That is very much not intentional. I will submit a PR for docs as well. I was unaware of this example being there.

Reply:
Yeah we've been running out of performance traces more often recently and were confused what had happened. And then I found that our sampler was not running correctly :-/

Reply:
Also, this is not all about the docs. It's like a public API and can't just be changed like this without proper deprecations etc. It must be costing your customers a lot of money in both extra Sentry charges as well as debugging time.

Reply:
I get what you're saying - Except it's not _my_ customers. I've done this as OSS contribution to help the Sentry Ruby SDK conform to the guidelines and "predefined operations" as defined by the Sentry core system - ie making the `Queues` page work properly for Sidekiq jobs.

Deprecating and semantic versioning of what goes into what release version is very not much my job.

Reply:
Aha I thought you worked for Sentry :)
Sorry if my tone was harsh, I didn’t mean to offend. I know mistakes can be made.
My message was more directed to Sentry the business.

Also the feature is great and many thanks for implementing it! 🙏